website/project/views/node.py

```diff
@@ -143,9 +143,6 @@ def folder_new_post(auth, node, **kwargs):
 
     title = request.json.get('title')
 
-    if not title or len(title) > 200:
```

Comment:
There were inconsistant/ random validations like throughout the code. Because this saves at the end, it runs validate_title and checks those two things already. Having that line would be redundant.